charts/sscs-evidence-share/values.template.yaml

```diff
@@ -0,0 +1,58 @@
+java:
+  image: ${IMAGE_NAME}
+  ingressHost: ${SERVICE_FQDN}
+  applicationPort: 8091
+  keyVaults:
+    sscs:
+      resourceGroup: sscs
+      secrets:
+        - idam-api
+        - idam-oauth-user
+        - idam-redirect
+        - idam-sscs-systemupdate-user
+        - idam-sscs-systemupdate-password
+        - idam-sscs-oauth2-client-secret
+        - ccd-api
+        - ccd-jid
+        - ccd-tid
+        - idam-s2s-api
+        - sscs-s2s-secret
+        - s2s-micro
+        - sscs-asb-primary-send-and-listen-shared-access-key
+#    docmosisiaasdevkv:
+#      excludeEnvironmentSuffix: true
+#      resourceGroup: docmosisiaasdevkv
+#      secrets:
+#        - docmosis-endpoint
+#        - docmosis-api-key
```

Comment:
note this didn't work. `docmosisiaasdevkv`  keyvault still could not be found.
However `PDF_SERVICE_ACCESS_KEY: ${PDF_SERVICE_ACCESS_KEY}` seems to work.